IFC 4.3/Alignment_3/UT_Alignment_3.ifc

```diff
@@ -17,45 +17,58 @@ DATA;
 #15= IFCSIUNIT(*,.VOLUMEUNIT.,$,.CUBIC_METRE.);
 #16= IFCSIUNIT(*,.PLANEANGLEUNIT.,$,.RADIAN.);
 #17= IFCSIUNIT(*,.TIMEUNIT.,$,.SECOND.);
-#18= IFCRELAGGREGATES('2IxncYFtr1GOR0ZHQutL4W',#10,$,$,#11,(#4));
-#19= IFCALIGNMENT('0ScqCY13TCGRKtSY7Hpgra',#10,'DefaultAlignment',$,$,#23,$,#42,$);
-#20= IFCRELCONTAINEDINSPATIALSTRUCTURE('1Z8B3uH5X5lQvcGxLyfi_2',#10,'Site','Site Container for Elements',(#19),#4);
+#18= IFCRELAGGREGATES('0FUwZVL557$x_gyGx$XvjR',#10,$,$,#11,(#4));
+#19= IFCALIGNMENT('01cry3QrfBlRQZCO0ltKjT',#10,'DefaultAlignment',$,$,#23,#55,#31,$);
+#20= IFCRELCONTAINEDINSPATIALSTRUCTURE('1VjJSy5zz4FeasG4yHriSi',#10,'Site','Site Container for Elements',(#19),#4);
 #21= IFCCARTESIANPOINT((0.0,0.0,0.0));
 #22= IFCAXIS2PLACEMENT3D(#21,$,$);
 #23= IFCLOCALPLACEMENT($,#22);
 #24= IFCCARTESIANPOINT((-29.29264,-6.90761));
 #25= IFCALIGNMENTHORIZONTALSEGMENT($,$,#24,0.462753315190078,0.0,0.0,72.3546021022319,$,.LINE.);
 #26= IFCDIRECTION((0.89482677,0.44641354));
-#27= IFCVECTOR(#26,1.0);
+#27= IFCVECTOR(#26,72.3546021022319);
 #28= IFCLINE(#24,#27);
 #29= IFCAXIS2PLACEMENT2D(#24,$);
-#30= IFCCURVESEGMENT(.CONTINUOUS.,#29,IFCNONNEGATIVELENGTHMEASURE(72.3546021022319),#28);
-#31= IFCALIGNMENTVERTICALSEGMENT($,$,0.0,72.349260880448,11.0,-0.0121508006435841,-0.0121508006435841,$,.CONSTANTGRADIENT.);
-#32= IFCDIRECTION((0.99992618,-0.0121505));
-#33= IFCVECTOR(#32,1.0);
-#34= IFCCARTESIANPOINT((72.34926,11.0));
-#35= IFCLINE(#34,#33);
-#36= IFCAXIS2PLACEMENT2D(#34,$);
-#37= IFCCURVESEGMENT(.CONTINUOUS.,#36,IFCNONNEGATIVELENGTHMEASURE(72.3546021022319),#35);
-#38= IFCALIGNMENTHORIZONTAL('2oAQ5oaBr0tOXC7B3AmcuP',#10,$,$,$,$,$,$,(#25));
-#39= IFCRELAGGREGATES('0AMRWI$Ar1xB77yIkOoXbF',#10,$,$,#19,(#38,#40));
-#40= IFCALIGNMENTVERTICAL('3BFgiHcZrDbxsfa4Y3noVU',#10,$,$,$,$,$,(#31));
-#41= IFCCOMPOSITECURVE((#30),.U.);
-#42= IFCGRADIENTCURVE(#41,(#37),$);
+#30= IFCCURVESEGMENT(.DISCONTINUOUS.,#29,IFCNONNEGATIVELENGTHMEASURE(72.3546021022319),#28);
+#31= IFCCOMPOSITECURVE((#30),.F.);
+#32= IFCALIGNMENTHORIZONTAL('0$gEmqHerCIO3weTxnESq9',#10,$,$,$,#43,#40,$,(#25));
+#33= IFCRELAGGREGATES('3URWl6MYP9Z8$_G9_d5iK_',#10,$,$,#19,(#32,#52));
+#34= IFCGEOMETRICREPRESENTATIONCONTEXT($,'Model',3,0.0001,#36,#37);
+#35= IFCCARTESIANPOINT((0.0,0.0,0.0));
+#36= IFCAXIS2PLACEMENT3D(#35,$,$);
+#37= IFCDIRECTION((0.0,1.0));
+#38= IFCGEOMETRICREPRESENTATIONSUBCONTEXT('Body','Model',*,*,*,*,#34,$,.MODEL_VIEW.,$);
+#39= IFCSHAPEREPRESENTATION(#38,'Body','Curve',(#31));
+#40= IFCPRODUCTDEFINITIONSHAPE($,$,(#39));
+#41= IFCCARTESIANPOINT((0.0,0.0,0.0));
+#42= IFCAXIS2PLACEMENT3D(#41,$,$);
+#43= IFCLOCALPLACEMENT($,#42);
+#44= IFCALIGNMENTVERTICALSEGMENT($,$,0.0,72.349260880448,11.0,-0.0121508006435841,-0.0121508006435841,$,.CONSTANTGRADIENT.);
+#45= IFCDIRECTION((0.99992618,-0.0121505));
+#46= IFCVECTOR(#45,72.3546021022319);
+#47= IFCCARTESIANPOINT((0.0,0.0));
+#48= IFCLINE(#47,#46);
+#49= IFCPOINTBYDISTANCEEXPRESSION(IFCNONNEGATIVELENGTHMEASURE(72.349260880448),$,$,$,#31);
```

Comment:
This would mean, that the segment *starts* at 72,34... which is probably not the case.

```suggestion
#49= IFCPOINTBYDISTANCEEXPRESSION(IFCNONNEGATIVELENGTHMEASURE(0.0),$,$,$,#31);
```